src/mon/FSCommands.cc

```diff
@@ -1525,6 +1513,30 @@ int FileSystemCommandHandler::_check_pool(
   }
 
   const string& pool_name = osd_map.get_pool_name(pool_id);
+  auto app_map = pool->application_metadata;
+
+  if(!allow_overlay && !app_map.empty()) {
+    if(app_map.find(pg_pool_t::APPLICATION_NAME_CEPHFS) != app_map.end()) {
+      auto& [app_name, app_metadata] = *app_map.find(pg_pool_t::APPLICATION_NAME_CEPHFS);
+      if(app_metadata.find("data") != app_metadata.end()) {
+	auto& [pool_type, filesystem] = *app_metadata.find("data");
+	*ss << "RADOS pool '" << pool_name << "' is already used by filesystem '"
+	  << filesystem << "' as a '" << pool_type << "' for application '"
+	  << app_name << "'";
```

Comment:
nit: just add the string **pool** after printing the `pool_type`
so you'll get to see **'data' pool for the application ...** or **'metadata' pool for the application ...**

Reply:
Done